sdk/core/core-client-rest/src/paginate.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="esnext.asynciterable" />
+
+import { Client, HttpResponse, PathUncheckedResponse } from "./";
+import "@azure/core-paging";
```

Comment:
this is to make sure the below type import doesn't get erased and we don't load the async iterator polyfill?

Reply:
I was following what other libraries did when using core-paging. I think it is just to load the polyfill before we try to import anything from core-paging. I'm removing it since we now support Node 10+ so the polyfill shouldn't be needed anymore